config.py

```diff
@@ -0,0 +1,11 @@
+class Config:
```

Comment:
Is this config specific to Nexus? Can we use a more specific class name that can make its more obvious?

Reply:
Rename to NexusConfig as it appears to be specific to you the download artifact from nexus functionality.

Reply:
Rename file from config.py to nexusconfig.py.

Reply:
why do these scripts need to be in common? can these live in common-tools?
You might also want to move these scripts under a package like confluent.tools or confluent.builds